# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../etcnet/0003-fix-use-of-VARPREFIX-in-sysctl-completion-code.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 2f8d7bc5e3acf2a75cee38a2c2d295687b897547 Mon Sep 17 00:00:00 2001
From: Sergey Bolshakov <sbolshakov@altlinux.org>
Date: Sun, 17 Jan 2010 20:00:38 +0300
Subject: [PATCH 3/4] fix use of VARPREFIX in sysctl completion code

---
 etc/net/scripts/ifup-common |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/etc/net/scripts/ifup-common b/etc/net/scripts/ifup-common
index 5b85b73..3570077 100755
--- a/etc/net/scripts/ifup-common
+++ b/etc/net/scripts/ifup-common
@@ -187,6 +187,8 @@ process_sysctl_conf()
 					print_error "Can't autocomplete sysctl variable name '$VARNAME' in file '$SRCFILE'. Please fix"
 					return 1
 			esac
+		else
+			VARPREFIX=
 		fi
 		$SYSCTL -q -w $VARPREFIX$VARNAME=$VARVALUE
 	done
-- 
1.6.6.2

